prac_08/silver_service_taxi_test.py

```diff
@@ -0,0 +1,10 @@
+"""
+Code for testing 'SilverServiceTaxi' class
+"""
+
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+taxi = SilverServiceTaxi('SSTaxi', 100, 2)
+taxi.drive(18)
+print(taxi)
+print(taxi.get_fare())
```

Comment:
Your tests should have something in them to show whether they pass or fail - like we did with Guitar expected/actual.